crates/turborepo-auth/src/sso_server.rs

```diff
@@ -22,10 +22,11 @@ pub struct SsoPayload {
 #[async_trait]
 pub trait SSOLoginServer {
     async fn run(&self, port: u16, verification_token: Arc<OnceCell<String>>) -> Result<(), Error>;
+    fn open_web_browser(&self, url: &str) -> std::io::Result<()>;
 }
 
-/// TODO: Document this.
-pub struct DefaultSSOLoginServer;
+/// Basic SSO login server. No configuration required.
+pub struct DefaultSSOLoginServer {}
```

Comment:
Keep construction simple
```suggestion
pub struct DefaultSSOLoginServer;
```